software/twenty.yml

```
@@ -0,0 +1,12 @@
+name: Twenty
+website_url: "https://twenty.com"
+source_code_url: "https://github.com/twentyhq/twenty"
+description: A modern CRM offering the flexibility of open source, advanced features, and a sleek design.
+licenses:
+  - AGPL-3.0
+platforms:
+  - TypeScript
+tags:
+  - Customer Relationship Management
```

New tags can only be created with 3 or more items. This can be discussed if there are others you think may fit. 

For now, this can go in Groupware, as this is where other CRMs are.

```suggestion
tags:
  - Groupware
```